debian/rules

```diff
@@ -151,6 +151,7 @@ DEPENDS += aptitude, \
 	   memstat, \
 	   mtr-tiny, \
 	   ncdu, \
+	   netcat, \
```

Comment:
I think the netcat package is actually called `netcat-openbsd`. The information on the plain `netcat` package says:

> This is a transitional package that depends on the netcat-openbsd
> implementation of netcat, preferred in Ubuntu.  It may be safely removed.